package.json

```diff
@@ -46,6 +46,7 @@
     "@testing-library/jest-native": "^5.4.2",
     "@testing-library/react-native": "^12.1.2",
     "@types/jest": "^29.5.2",
+    "@types/react-native": "^0.72.2",
```

Comment:
Do we really need this? I thought React Native is already typed

Reply:
actually we dont